percy/percy-web
# Upgrade ember-data, ember-source, ember-cli

### Blocked on: 
- ~https://github.com/percy/percy-api/pull/394~
- ~https://github.com/percy/percy-api/pull/396~


QA:
- PR for this: https://github.com/percy/percy-api/pull/396 ~error when creating new project for org: creates request: `POST http://dev.percy.local:4200/api/v1/organizations/org-slug/projects?` -- error: `{status: "bad_request", detail: "param is missing or the value is empty: repo"}` -- looks like there is a problem with the url formatting-- there is an unnecessary question mark?
This is the error, but it seems to actually break when you don't include a repo. It's not related to strong params.~
~Removing the requirement for a github repo fixes this problem.~
- Can no longer reproduce this... ~Some combination of these steps makes the org list in the org dropdown disappear~
    - ~Go to org settings page. Change name or slug of organization. The page will reload, and forward you to the org index. Go back to org settings page, open org dropdown. Orgs are gone.~
- No more cancel subscription button, it is not used ~Cancel subscription breaks? does this work on production?~
